README.md

```diff
@@ -1,112 +1,649 @@
+  "Accept" : "application/json",
+  "Authorization" : "Bearer {clienteToken}"
+}
+</pre>
+    </td>
+  </tr>
+  <tr>
+    <td>
+<pre>
+Body
+{
+  "cpf": "string",
+  "senha": "string"
+}
+</pre>
+    </td>
+  </tr>
+<tr></tr>
+  <tr>
+    <td colspan="3" align="center">  :arrow_right_hook: Cliente faz um deposito em uma conta, seja dele ou não </td>
+  </tr>
+  <tr>
+    <td rowspan="2">
+      /api/clientes/{idCliente}/deposito
+    </td>
+    <td rowspan="2">
+      POST
+    </td>
+    <td>
+<pre>
+Header
+{
+  "Content-Type" : "application/json",
+  "Accept" : "application/json",
+  "Authorization" : "Bearer {clienteToken}"
+}
+</pre>
+    </td>
+  </tr>
+  <tr>
+    <td>
+<pre>
+Body
+{
+  "numeroContaDestino" : "string",
+  "valor" : 10
+}
+</pre>
+    </td>
+  </tr>
+<tr></tr>
+  <tr>
+    <td colspan="3" align="center">  :arrow_right_hook: Cadastra uma nova transação entre contas </td>
+  </tr>
+  <tr>
+    <td rowspan="2">
+      /api/clientes/{idCliente}/transacao
+    </td>
+    <td rowspan="2">
+      POST
+    </td>
+    <td>
+<pre>
+Header
+{
+  "Content-Type" : "application/json",
+  "Accept" : "application/json",
+  "Authorization" : "Bearer {clienteToken}"
+}
+</pre>
+    </td>
+  </tr>
+  <tr>
+    <td>
+<pre>
+Body
+{
+  "dataEnvio": "2021-12-04T20:16:11.211Z",
+  "dataRecebimento": "2021-12-04T20:16:11.211Z",
+  "idContaEmissora": 2,
+  "idContaReceptora": 1,
+  "senhaContaEmissora": "string",
+  "valor": 10
+}
+</pre>
+    </td>
+  </tr>
+<tr></tr>
+  <tr>
+    <td colspan="3" align="center">  :arrow_right_hook: Ver historico de transações de um cliente </td>
+  </tr>
+  <tr>
+    <td rowspan="2">
+      /api/clientes/{idCliente}/transacoes
+    </td>
+    <td rowspan="2">
+      GET
+    </td>
+    <td>
+<pre>
+Header
+{
+  "Accept" : "application/json",
+  "Authorization" : "Bearer {clienteToken}"
+}
+</pre>
+    </td>
+  </tr>
+  <tr>
+    <td>
+<pre>
+Body
+{
+}
+</pre>
+    </td>
+  </tr>
+<tr></tr>
+  <tr>
+    <td colspan="3" align="center">  :arrow_right_hook: Cadastra um email de administrador como uma subscription no SNS da AWS para receber notificação de cadastro de novos clientes. </td>
+  </tr>
+  <tr>
+    <td rowspan="2">
+      /api/sns/subscricao/{email}
+    </td>
+    <td rowspan="2">
+      POST
+    </td>
+    <td>
+<pre>
+Header
+{
+  "Accept" : "application/json"
+}
+</pre>
+    </td>
+  </tr>
+  <tr>
+    <td>
+<pre>
+Body
+{
+}
+</pre>
+    </td>
+  </tr>
+<tr></tr>
+</table>
+
+<hr>
+
+<h3 id="aws-swagger"> 🔃 Deploy AWS e links do Swagger UI</h3>
+
+- <h4 id="deploy-ec2" >Deploy AWS EC2:</h4>
+
+  - <a href="./assets/step-by-step/deploy-ec2-aws.md"> 🔗 Passo a passo</a>
+  - URL: 🔗 http://ec2-52-23-240-140.compute-1.amazonaws.com:8080/
+  - <a href="http://ec2-52-23-240-140.compute-1.amazonaws.com:8080/swagger-ui.html"> 🔗 Link Swagger UI</a>
+
+- <h4 id="deploy-bs" >Deploy AWS Beanstalk:</h4>
+
+  - <a href="./assets/step-by-step/deploy-ec2-bs.md"> 🔗 Passo a passo</a>
```

Comment:
Page not found neste link de passo a passo

Reply:
Esse ai vai ser um arquivo que eu acho que a @ayanara ainda vai criar , mas foi bom vc marcar pq já percebi que deixei o nome errado, seria apenas ./assets/step-by-step/deploy-bs.md